localstack/utils/container_utils/container_client.py

```diff
@@ -103,6 +105,23 @@ class DockerPlatform(str):
     linux_arm64 = "linux/arm64"
 
 
+class Ulimit(TypedDict, total=False):
```

Comment:
I guess we aim to keep the `container_client` generic and therefore added a `Ulimit` type instead of importing `docker.types.Ulimit`